app/models/passenger.rb

```diff
@@ -34,13 +34,4 @@ def expiration_display
   def temporary?
     !permanent?
   end
-
-  def self.deactivate_expired_doc_note
-    expired = active.joins(:doctors_note)
-                    .where('doctors_notes.expiration_date < ?',
-                           DoctorsNote.grace_period)
-    expired.each do |passenger|
-      passenger.update_attributes active: false
-    end
-  end
```

Comment:
Are we no longer letting the database decide who is active based on doctor's notes?

Reply:
There's no reason for a database attribute, no. The "inactive" people are highlighted on the index page just like before. I didn't see any reason to have a database attribute, although if you can see one I'm all ears.